wiki/Tournaments/OWC/1/en.md

```diff
@@ -1,152 +1,165 @@
-| ![][flag_DO] Dominican Republic | **Lissette**, Lizbeth, jmt_3, Magestix                                 |
-| ![][flag_US] United States      | **Lybydose**, Mafiamaster, Cyclone (ebacho, naptime)                   |
-
-Map list
----------
-
-Win condition: Team with highest total team score wins the match.
-
-| Round                | Advance rule                |
-|----------------------|-----------------------------|
-| \#1-\#3              | Team win by 2-0             |
-| Semifinal, 3rd place | First team to win 4 matches |
-| Final                | First team to win 5 matches |
-
-| Round                | Beatmap rule                                                  |
-|----------------------|---------------------------------------------------------------|
-| \#1-\#3              | One beatmap from the 15 beatmaps per team in respective week. |
-| Semifinal, 3rd place | Predetermined list of 6 beatmaps in order.                    |
-| Final                | Predetermined list of 8 beatmaps in order.                    |
-
-| Round                | Draw case                    |
-|----------------------|------------------------------|
-| \#1-\#3              | Predetermined beatmap. (1-1) |
-| Semifinal, 3rd place | #7 beatmap (3-3)             |
-| Final                | #9 beatmap (4-4)             |
-
-### \#1st week
-
-
-- Pick one beatmap
+In every World Cup, people conquering a place on the podium are eligible to receive unique prizes created specifically for World Cup winners. It is possible that these items change with every installment of the World Cups.
+
+| Place | Prize(s) |
+| ----- | -------- |
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place")   | 6 months supporter tag, special profile badge (![OWC 1 Profile Badge](badge.png)) |
+| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | 1 month supporter tag |
+| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | 1 month supporter tag |
+| *Special* | 1 month supporter tag (won by ![][flag_JP] Team Japan) |
+
+## Organization
+
+The osu! World Cup #1 is run by various community members by distributing the multitude of tasks into various fields of responsibility.
+
+- ![][flag_TW] [Alace](https://osu.ppy.sh/users/25993)
+- ![][flag_ES] Beuchi-chan
+- ![][flag_US] [Derekku](https://osu.ppy.sh/users/91341)
+- ![][flag_AU] [m980](https://osu.ppy.sh/users/3288)
+- ![][flag_DE] [Larto](https://osu.ppy.sh/users/12328)
+- ![][flag_JP] [dvorak](https://osu.ppy.sh/users/271359)
+- ![][flag_IT] [Lunah](https://osu.ppy.sh/users/1227377)
+
+## Links
+
+- [Discussion thread](https://osu.ppy.sh/community/forums/topics/45412)
+- [Match recaps](https://osu.ppy.sh/community/forums/topics/45412?n=803)
+- [Finals recap](https://www.mediafire.com/file/ab6j6k4ihtp25o2) - provided by [dvorak](https://osu.ppy.sh/users/271359)
+
+------------------------------------------------------------
+
+## Participants
+
+### Confirmed Rosters
+
+| Country | Team Members |
```

Comment:
Its better to do an extra column at the start just for the flags, then center and bold the country names.
For the profile links, i think we can do that work later, many users changed their username or they are restricted.